pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "core-ct"
+version = "0.1.0"
+description = "A Python library to assist geologists with the analysis of rock core CT scans"
+authors = ["Your Name <you@example.com>"]
```

Comment:
we should put our names and emails here

Reply:
I definitely meant to add them earlier but I totally forgot. Thanks for the reminder! I just updated the most recent commit to include them